pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -56,7 +56,7 @@ func ServeVMIPreset(resp http.ResponseWriter, req *http.Request) {
 }
 
 func ServeMigrationCreate(resp http.ResponseWriter, req *http.Request, virtCli kubecli.KubevirtClient) {
-	validating_webhooks.Serve(resp, req, admitters.NewMigrationCreateAdmitter(virtCli))
+	validating_webhooks.Serve(resp, req, admitters.NewMigrationCreateAdmitter(virtCli.GeneratedKubeVirtClient()))
```

Comment:
I see this is also done in https://github.com/kubevirt/kubevirt/pull/11897.

Is there a reason not to change `virtAPIApp`'s virtCli to the generated client in general, hence do it for all webhooks altogether?

Reply:
Since there are unit test using gomock, this is not a trivial change.
It requires change also to the unit tests.